### PR TITLE
Make tell thread-safe by adding locks everywhere

### PR DESCRIPTION
**Pull request by nyuszika7h**
Ported from willie.

Ref.: https://github.com/embolalia/willie/commit/858e7ec3efac0e8ff17cce4e4148645473bcb832

### CODE REVIEW

File: modules/tell.py [hunk 1]
```diff
@@ -11,33 +11,41 @@
...
+    lock.acquire()
+    try:
+        result = {}
+        f = open(fn)
+        for line in f:
+            line = line.strip()
+            if line:
+                try: tellee, teller, verb, timenow, msg = line.split('\t', 4)
+                except ValueError: continue  # @@ hmm
+                result.setdefault(tellee, []).append((teller, verb, timenow, msg))
```
Comment (kaneda) [inline modules/tell.py]: If this blows up in the except above the values appended will be None type, does that screw anything up?

**Comment by kaneda** [comment]
One question, otherwise :shipit:

File: modules/tell.py (hunk 1 above)
Comment (nyuszika7h) [inline modules/tell.py]: No idea, but that code was already there before my change.

**Comment by kaneda** [comment]
@myano can I get your eyeballs on this one?

**Pull request merged**